save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/Execution.kt

```diff
@@ -60,8 +59,6 @@ class Execution(
     @Enumerated(EnumType.STRING)
     var status: ExecutionStatus,
 
-    var testSuiteIds: String?,
```

Comment:
what's about `@ManyToMany` and add List of TestSuite here?

Reply:
I thought about it, but decided that it is not necessary. It is rarely used. And the LnkExecutionTestSuite does not store any information that could be passed to the executionDTO for transmission to the front.